docs/events/api/timeline-template-api-update.asciidoc

```diff
@@ -0,0 +1,385 @@
+      "filterQuery": null
+    },
+    "title": "abd",
+    "dateRange": {
+      "start": 1587370079200,
+      "end": 1587456479201
+    },
+    "savedQueryId": null,
+    "sort": {
+      "columnId": "@timestamp",
+      "sortDirection": "desc"
+    },
+    "created": 1587468588922,
+    "createdBy": "casetester",
+    "updated": 1587468588922,
+    "updatedBy": "casetester",
+    "timelineType": "default"
+  },
+  "timelineId": "4bc294e0-3516-11ee-9f62-49614d8a84fd", <1>
+  "version": "WzE5MTUsMV0=" <2>
+}
+--------------------------------------------------
+
+<1> `timelineId` must match the existing Timeline's `savedObjectId`.
+<2> `version` must match the existing Timeline version.
+ 
+*Example 2*
+
+Updates an existing Timeline template.
+
+[source,console]
+--------------------------------------------------
+PATCH api/timeline
+{
+  "timeline": {
+    "columns": [
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "@timestamp"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "message"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "event.category"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "event.action"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "host.name"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "source.ip"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "destination.ip"
+      },
+      {
+        "columnHeaderType": "not-filtered",
+        "id": "user.name"
+      }
+    ],
+    "dataProviders": [],
+    "description": "",
+    "eventType": "all",
+    "filters": [],
+    "kqlMode": "filter",
+    "kqlQuery": {
+      "filterQuery": null
+    },
+    "title": "abd",
+    "dateRange": {
+      "start": 1587370079200,
+      "end": 1587456479201
+    },
+    "savedQueryId": null,
+    "sort": {
+      "columnId": "@timestamp",
+      "sortDirection": "desc"
+    },
+    "timelineType": "template",
+    "created": 1587473119992,
+    "createdBy": "casetester",
+    "updated": 1587473119992,
+    "updatedBy": "casetester",
+    "templateTimelineId": "6f9a3480-bf4f-11ea-9fcd-ed4e5fd0dcd1", <1>
+    "templateTimelineVersion": 2 <2>
+  },
+  "timelineId": "7d7d4b60-3516-11ee-9f62-49614d8a84fd", <3>
+  "version": "WzE5MTcsMV0=" <4>
+}
+--------------------------------------------------
+<1> `templateTimelineId` must match the existing Timeline template's `templateTimelineId`. 
+<2> `templateTimelineVersion` must be a Timeline template version greater than the existing one.
+<3> `timelineId` must match the existing Timeline's `savedObjectId`.
+<4> `version` must match the existing Timeline version.
+
+==== Response code
+
+`200`::
+    Indicates a successful call.
+
+==== Response payload
+
+A JSON Timeline object with a unique `savedObjectId` and its `version`.
+
+*Example 1*
+
+Update Timeline response payload:
```

Comment:
Suggesting some minor tweaks for clarity here - ofc the edits are totally optional.

```suggestion
Response payload of an updated Timeline:
```